utils/swift_build_support/swift_build_support/cmake.py

```diff
@@ -119,6 +119,8 @@ def common_options(self):
 
         if args.build_ninja and args.cmake_generator == 'Ninja':
             define('CMAKE_MAKE_PROGRAM', toolchain.ninja)
```

Comment:
I think that matches the principle of "common build-script tasks should be short". Of course, we could just say Ninja is a requirement for the Swift project now and drop support for building it altogether, but that's a little annoying for Mac users who wouldn't otherwise be installing it somewhere.